scripts/news/utils.py

```diff
@@ -0,0 +1,196 @@
+import base64
+import datetime
+import glob
+import hashlib
+import os
+import sys
+import textwrap
+import traceback
+from pathlib import Path
+from typing import Any, Dict, List, Mapping, Optional, Tuple, Union
+
+import tomli
+from click import Context, Option, echo, style
+from jinja2 import Template
+
+from . import ERROR_MSG_PREFIX
+
+
+def nonceify(body: str) -> str:
+    """
+    Nonceify the news body!
+
+    Generate hopefully-unique string of characters meant to prevent filename collisions. by computing the
```

Comment:
It will "likely" have one change, I have seen several PRs out of this repo which includes more than 1 major change, hence I have added support for this. I am fine with incrementing but it's fine with this string also, it doesn't really matter IMO

Reply:
incrementing will tell the user which should go first, and that allows the order to change as well.